rpmlint/descriptions/SUIDPermissionsCheck.toml

```diff
@@ -0,0 +1,104 @@
+permissions-unauthorized-file="""
+If the package is intended for inclusion in any SUSE product please open a bug
+report to request review of the package by the security team. Please refer to
+https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs for
+more information.
+"""
+
+
+
+permissions-symlink="""
+permissions handling for symlinks is useless. Please contact security@suse.de
+to remove the entry. Please refer to
+https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs for
+more information.
+"""
+
+
+permissions-dir-without-slash="""
+the entry in the permissions file refers to a directory. Please contact
+security@suse.de to append a slash to the entry in order to avoid security
+problems. Please refer to
+https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs for
+more information.
+"""
+
+
+permissions-file-as-dir="""
+the entry in the permissions file refers to a directory but the package
+actually contains a file. Please contact security@suse.de to remove the slash.
+Please refer to
+https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs for
+more information.
+"""
+
+
+permissions-incorrect="""
+please use the %attr macro to set the correct permissions.
+"""
+
+
+permissions-incorrect-owner="""
+please use the %attr macro to set the correct ownership.
+"""
+
+
+permissions-file-setuid-bit="""
+If the package is intended for inclusion in any SUSE product please open a bug
+report to request review of the program by the security team. Please refer to
+https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs for
+more information.
+"""
+
+
+permissions-directory-setuid-bit="""
+If the package is intended for inclusion in any SUSE product please open a bug
+report to request review of the package by the security team. Please refer to
+https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs for
+more information.
+"""
+
+
+permissions-world-writable="""
+If the package is intended for inclusion in any SUSE product please open a bug
+report to request review of the package by the security team. Please refer to
+https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs for
+more information.
+"""
+
+
+permissions-fscaps="""
+Packaging file capabilities is currently not supported. Please use normal
+permissions instead. You may contact the security team to request an entry that
+sets capabilities in /usr/share/permissions/permissions instead.
+"""
+
+
+permissions-missing-postin="""
+Please add an appropriate %post section
+"""
+
+
+permissions-missing-requires="""
+Please add 'PreReq: permissions'
```

Comment:
PreReq is obsolete variable.

Reply:
Should be `Requires` used here?

Reply:
No Requires(phase): one.

Reply:
I think it should be a plain `Requires`, because the `verifyscript` also has that dependency.